appeals/web/src/server/appeals/appeal-details/service-user/service-user.mapper.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @typedef {import('../appeal-details.types.js').WebAppeal} Appeal
+ */
+
+import { appealShortReference } from '#lib/appeals-formatter.js';
+import {
+	errorEmail,
+	errorFirstName,
+	errorLastName
+} from '#lib/error-handlers/change-screen-error-handlers.js';
+
+/**
+ *
+ * @param {Appeal} appealData
+ * @param {string} userType
+ * @param {import('../appeal-details.types.js').WebServiceUser} userDetailsInSession
+ * @param {import('@pins/express').ValidationErrors | undefined } errors
+ * @returns {PageContent}
+ */
+export const changeServiceUserPage = (appealData, userDetailsInSession, userType, errors) => {
+	// @ts-ignore
+	let serviceUserDetails = appealData[userType];
```

Comment:
As mentioned in a different comment, ideally the `ignore` wouldn't be necessary, is there any way to work around the TS issue here?

Reply:
Because the userType is of type string but it is validated in the get to be a valid key of appealDetails. Problem is that using that check doesn't fix the TS error and I want to avoid listing the valid userTypes as more are coming (could be something to add once we have all the valid user types and data models)